cypress/e2e/tests/migration/task-manager/manage_columns.test.ts

```diff
@@ -37,10 +36,6 @@ describe(["@tier3"], "Task manager - table column management validation", functi
         TaskManagerTableHeaders.terminated,
     ];
 
-    before("Login", function () {
-        login();
```

Comment:
question: Why are we removing the `login` calls in some tests and leaving them in others?

Reply:
If there is only login() in before method , it works without it as well because there is a login() in global ,BeforeEach().
But if there are some other UI steps after login() in before () method then it needs to be followed by cy.isit("/") as before is called before() global beforeEach()